<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>545</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>401</width>
    <height>545</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>401</width>
    <height>545</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RTT2UART Control Panel</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pushButton_Start">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>500</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>225</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Target Interface And Speed</string>
   </property>
   <widget class="QComboBox" name="comboBox_Interface">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Speed">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Specify Target Device</string>
   </property>
   <widget class="QComboBox" name="comboBox_Device">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Selete_Device">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_UART">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>328</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>UART Config</string>
   </property>
   <widget class="QComboBox" name="comboBox_Port">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>48</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Baud rate:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_baudrate">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>48</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scan">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox_resettarget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset target</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_log_split">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Split</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_RTTControlBlock">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>381</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>RTT Control Block</string>
   </property>
   <widget class="QRadioButton" name="radioButton_AutoDetection">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Detection</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Address">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_SearchRange">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Range</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_RTTAddress">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>48</y>
      <width>358</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter the address of the RTT control block</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Connection to J-Link</string>
   </property>
   <widget class="QRadioButton" name="radioButton_usb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>USB</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_existing">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Existing Session</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_tcpip">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP/IP</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_serialno">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial NO</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_serialno">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>18</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh_jlink">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>18</y>
      <width>26</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>↻</string>
    </property>
    <property name="toolTip">
     <string>Refresh JLink devices</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ip">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox__auto">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Reconnect</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_SerialForward">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>381</width>
     <height>75</height>
    </rect>
   </property>
   <property name="title">
    <string>Serial Forward Settings</string>
   </property>
   <widget class="QRadioButton" name="radioButton_LOG">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LOG Current Tab Selection</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_DATA">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DATA (RTT Channel 1)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_SerialForward">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>45</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Forward Content:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_SerialForward">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>43</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>510</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
